cloudflare/data_source_waf_packages.go

```diff
@@ -0,0 +1,182 @@
+package cloudflare
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"time"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+func dataSourceCloudflareWAFPackages() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceCloudflareWAFPackagesRead,
+
+		Schema: map[string]*schema.Schema{
+			"zone_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
```

Comment:
This field is required, yet not documented.

Reply:
Fixed!